lib/topological_inventory/openshift/operations/core/service_catalog_client.rb

```diff
@@ -115,6 +121,15 @@ def fetch_authentication
             auth_id = endpoint_authentications.first.id
             AuthenticationRetriever.new(auth_id, identity).process
           end
+
+          def api_client
```

Comment:
Where are you using api_client?  I'd rather not define this same method multiple times...we should pull this out into its own module